Dockerfile

```diff
@@ -133,6 +133,10 @@ RUN CGO_ENABLED=0 go build -mod=vendor -o /bin/pfcpiface
 
 # Stage pfcpiface: runtime image of pfcpiface toward SMF/SPGW-C
 FROM alpine AS pfcpiface
+COPY conf /opt/bess/bessctl/conf
```

Comment:
Are we using /opt/bess/bessctl/conf directory contents anywhere in the code?

Reply:
Will remove.

Reply:
still need to be removed?

Reply:
Needs to be present. upf.json is accessed from this path.